src/internal/privateTypes.ts

```diff
@@ -80,8 +71,7 @@ export interface DeviceInfoNativeModule {
   getIncrementalSync: () => string;
   isPinOrFingerprintSet: () => Promise<boolean>;
   isPinOrFingerprintSetSync: () => boolean;
-  hasNotch: () => Promise<boolean>;
-  hasNotchSync: () => boolean;
+  hasNotch: () => boolean;
```

Comment:
will this fix the chrome debugger issue? Seems like we are still calling `hasNotch` synchronously, why not make it a constant? Sorry if I've missed something, native code is not my zone usually.

Reply:
It's subtle, but hasNotch is both sync here and not a problem, because it doesn't actually touch native code, so it won't cause problems with the chrome debugger. hasNotch in this module is implemented entirely in javascript as a lookup table

Reply:
@mikehardy ahh thanks for the explanation! And the hard work 😄

Reply:
you're welcome - and definitely keep poking around on the PRs - an extra set of eyes always helps